playwright/src/main/java/com/microsoft/playwright/junit/Options.java

```diff
@@ -0,0 +1,112 @@
+package com.microsoft.playwright.junit;
+
+import com.microsoft.playwright.APIRequest;
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.BrowserType;
+import com.microsoft.playwright.Playwright;
+import com.microsoft.playwright.options.ViewportSize;
+
+import java.nio.file.Path;
+
+public class Options {
+  public String baseUrl;
+  public Path storageStatePath;
+  public ViewportSize viewportSize;
+  public String channel;
+  public Boolean headless;
+  public String browserName = "chromium";
+  public BrowserType.LaunchOptions launchOptions;
+  public Browser.NewContextOptions contextOption;
+  public APIRequest.NewContextOptions apiRequestOptions;
+  public Playwright.CreateOptions playwrightCreateOptions;
+
+  public Playwright.CreateOptions getPlaywrightCreateOptions() {
+    return playwrightCreateOptions;
+  }
+
+  public Options setPlaywrightCreateOptions(Playwright.CreateOptions playwrightCreateOptions) {
```

Comment:
@yury-s should we remove all the setters from the Options class?  I don't see them being useful since the user would only override the getters, correct?